test/drenv/minio.py

```diff
@@ -50,14 +50,13 @@ def _service_info(cluster):
     return host_ip, service_port
 
 
-def _service_is_alive(host_ip, service_port, log=print):
+def _service_is_ready(host_ip, service_port, log=print):
     """
-    Based on
-    https://min.io/docs/minio/linux/operations/monitoring/healthcheck-probe.html#node-liveness
+    Based on https://github.com/minio/minio/issues/19412#issuecomment-2037702457
     """
     conn = http.client.HTTPConnection(host_ip, service_port, timeout=60)
     try:
-        conn.request("HEAD", "/minio/health/live")
+        conn.request("HEAD", "/minio/health/ready")
         r = conn.getresponse()
         if r.status != http.HTTPStatus.OK:
             log(f"Service returned unexpected status code: {r.status}")
```

Comment:
We can replace this function with call to `mc ready`:
https://github.com/minio/minio/issues/19412#issuecomment-2038128138

Reply:
There is no timeout in `mc ready` so I don't think we can use it yet.